code/modules/mob/living/carbon/taste.dm

```diff
@@ -20,7 +20,7 @@ calculate text size per text.
 	var/minimum_percent = 15
 	if(ishuman(taster))
 		var/mob/living/carbon/human/H = taster
-		minimum_percent = round(15/ (H.isSynthetic() ? TASTE_DULL : H.species.taste_sensitivity))
```

Comment:
I dont think this is correct, you probably wouldn't use round on a hardcoded number